### PR TITLE
ci: disable non-functional CI

### PR DESCRIPTION
**Pull request by avik-pal**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -72,76 +72,76 @@ steps:
...
-    steps:
-      - label: "CPU: Run Benchmarks"
-        plugins:
-          - JuliaCI/julia#v1:
-              version: "1"
-        command: |
-          julia --project=benchmark -e 'println("--- :julia: Instantiating project")
-              using Pkg
-              Pkg.develop([PackageSpec(path=pwd()), PackageSpec(path="lib/ReactantCore")])'
+  # - group: ":racehorse: Benchmarks"
```
Comment (avik-pal) [inline .buildkite/pipeline.yml:75]: I will fix the CI in a different PR

**Comment by avik-pal** [comment]
Docs CI is fixed in the jll bump PR

**Comment by mofeing** [comment]
I was thinking about this the other day, so I'm fully in favor

**Comment by avik-pal** [comment]
We discussed this in yesterday's meeting and there was general approval so I am merging this

**Pull request merged**